Clean updated Docker topics for Docker-1.0.0 (#6406)
magento/devdocs: src/cloud/docker/docker-mftf.md

  - Cloud
---

In a Cloud Docker development environment, you can use the [Magento Functional Testing Framework (MFTF)][MFTF docs] for Magento application testing.

> You use the [CLI container command] `mftf-command` to run MFTF commands in the Cloud Docker environment, for example `docker-compose run test mftf-command generate:tests --debug=none`.

Reviewer: This is good to have! Could use a little more comment about what the example is doing, though.

> You use the [CLI container command] `mftf-command` to run MFTF commands in the Cloud Docker environment, for example `docker-compose run test mftf-command generate:tests --debug=none`.

Author: ```suggestion
You use the [CLI container command] `mftf-command` to run MFTF commands in the Cloud Docker environment. For example, the following command generates the MFTF tests: 

```bash
docker-compose run test mftf-command generate:tests --debug=none`.
```

  - Cloud
---

In a Cloud Docker development environment, you can use the [Magento Functional Testing Framework (MFTF)][MFTF docs] for Magento application testing.
In this environment, you run MFTF commands using the [CLI container command] `mftf-command`. For example, the following command generates the MFTF tests:

```bash
docker-compose run test mftf-command generate:tests --debug=none
```

{:.bs-callout-info}
Support for MFTF requires [Magento Cloud Docker] version 1.0 or later.

> 1. Prepare the local environment:

Reviewer: ```suggestion
1. Prepare the local environment.
```

    ./vendor/bin/ece-docker build:compose --with-selenium --no-cron
    ```

1. Start the Magento Cloud Docker environment.

    Optionally, you can set up {{site.data.var.mcd}} to work in [Developer Mode].

>     ```bash

Reviewer: There are two commands here. For copy purposes, may need to be split.

    ./vendor/bin/ece-docker build:compose --with-selenium --no-cron
    ```

1. Start the Magento Cloud Docker environment. Optionally, you can set up {{site.data.var.mcd}} to work in [Developer Mode].

   ```bash
   ./bin/magento-docker up
   ```

   ```bash
    ./bin/magento-docker ece-redeploy
    ```

> 1. Prepare the Magento application by adding environment variables that are specific to MFTF:

Reviewer: ```suggestion
1. Prepare the Magento application by adding environment variables that are specific to MFTF.
```

   ```bash
   CONFIG="MAGENTO_BASE_URL=http://magento2.docker/
   MAGENTO_BACKEND_NAME=admin
   MAGENTO_ADMIN_USERNAME=admin
   MAGENTO_ADMIN_PASSWORD=123123q
   MODULE_WHITELIST=Magento_Framework,Magento_ConfigurableProductWishlist,Magento_ConfigurableProductCatalogSearch
   SELENIUM_HOST=selenium"
   ```

   ```bash
   docker-compose run deploy bash -c "echo \"$CONFIG\" > /app/dev/tests/acceptance/.env"
   ```

   {:.bs-callout-tip}

>    In this example, the variable configuration is for testing a Magento application deployed to the Docker environment. To run tests in a remot environment, change the value of `MAGENTO_BASE_URL` to the remote URL and update the credentials as needed.

Author: ```suggestion
   In this example, the variable configuration is for testing a Magento application deployed to the Docker environment. To run tests in a remote environment, change the value of `MAGENTO_BASE_URL` to the remote URL and update the credentials as needed.
```

>       {:.bs-callout-info}

Reviewer: I do not like this step at all. I think it is very difficult to follow on screen. I already spent a lot of time reformatting this file and sent to you, so you have my suggestion for it already.
